README.md

```diff
@@ -31,4 +31,12 @@ Accessing other database systems is possible, with a reduced feature set.
 The [manual and scaladocs](http://slick.lightbend.com/docs/) for Slick can be
 found on the [Slick web site](http://slick.lightbend.com/).
 
-Licensing conditions (BSD-style) can be found in LICENSE.txt.
+## Maintenance status
+
+Slick is community-maintained.  The lead volunteer maintainer is currently
+@hvesalai.  Lightbend staff (such as @SethTisue) may be able to assist
+with administrative issues.
+
+Slick is no longer covered under Lightbend commercial support.
```

Comment:
Current LB customers are still covered until Dec 31, 2021. Not sure we need the last paragraph here (documentation duplication). Customers were informed last year.

Reply:
Okay, I dropped the second paragraph.